tests/cli/test_utils.py

```diff
@@ -513,6 +513,17 @@ def test_list_instances_no_service(
     assert actual == expected
 
 
+def test_validate_service_instance_gives_suggestions():
```

Comment:
Should be deleted in favor of the tests you added in tests/test_utils.py, right? It's broken anyway.

Reply:
Whoa whoops! 
Nothing to see here....